docs/docs/recipes/manage-users/on-admin-console-page.md

```diff
@@ -7,7 +7,18 @@ sidebar_position: 1
 
 ## Add user
 
-TODO: admin manually add
+Not only can the end-user sign up by his/herself, but also the admin users can manually add accounts for them.
+
+1. In the "User Management" panel, click the "+ Add User" button at the top right corner.
+2. In the "Add User" dialog, fill out the "Username" (for sign-in) and "Full name" input boxes, and click the "Add User" button to confirm.
+3. After successfully adding a new user, you can see their username and initial password in the dialog.
+   Click the eye icon following the password to show or hide the actual password.
+4. Click the "Copy" button to copy the **username** and **initial password** to the clipboard so that you can send it to the user to sign in.
```

Comment:
Too many details?  We will have multiple screenshots for different steps.

Reply:
Will do it later.